frontend/projects/ui/src/app/app/global/services/patch-monitor.service.ts

```diff
@@ -14,7 +13,7 @@ export class PatchMonitorService extends Observable<boolean> {
   private readonly stream$ = this.authService.isVerified$.pipe(
     switchMap(verified => {
       if (verified) {
-        return from(this.patch.start()).pipe(mapTo(true))
+        return from(this.patch.start()).pipe(map(() => true))
```

Comment:
There doesn't seem to be anything async left in `patch.start`. Looks like we can remove `async` from it and just use `map` instead of `switchMap` here.